### ISSUE TITLE
Documents for AMS-LaTeX classes and IEEEtran

### ISSUE THREAD

**Issue by wtsnjp** [TeX-Live/texdoc#57]
I wrote a small script to check the aliases in our [`texdoc.cnf`](https://github.com/TeX-Live/texdoc/blob/master/texdoc.cnf) and list up all not working aliases. I found 26 invalid aliases in the file and fixed or removed most of them. But there are two cases which I cannot find good solutions and would like to ask for your help.

1. AMS-LaTeX classes

For the name of AMS-LaTeX classes, namely `amsart`, `amsbook`, `amscls`, `amsproc`, they were aliased to `instr-l.pdf`. However, this file seems no longer exists in TeX Live.

There is a document `texmf-dist/doc/latex/amscls/amsclass.pdf` in the current TeX Live, but this looks like a source code documentation and not for normal users. Which document is preferred to show with these names? Do you have any suggestions?

2. IEEEtran

<s>For the name `ieee`, it was aliased to `IEEEtran_HOWTO.pdf`. However, it seems this file also not exist in the current TeX Live though the package IEEEtran exists (I tried the `find` command under the `texmf-dist` but not found). Is there the file or an alternative in TeX Live?</s>

<s>JFYI, there is a TeX Live-specific README (`texmf-dist/doc/latex/IEEEtran/README.TEXLIVE`) and it says:</s>

```
The following files have been removed in the TeX Live installation of
the current package, typically due to duplication, lack of space, or
missing source code.  You can find these files on CTAN at
	http://mirror.ctan.org/macros/latex/contrib/IEEEtran
If questions or concerns, email tex-live@tug.org.

	IEEEtran_bst_HOWTO.pdf
	IEEEtran_HOWTO.pdf
```

<s>If the file `IEEEtran_HOWTO.pdf` doesn't exist in TeX Live, I think this file also needs to be updated.</s>

(I didn't read the readme carefully, sorry.)

Which document is preferred to show with the name "IEEE"? Do you have any suggestions?

**Comment by ballapete** [comment]
> Am 24.4.2020 um 11:18 schrieb Takuto ASAKURA <notifications@github.com>:
> 
> I wrote a small script to check the aliases in our texdoc.cnf and list up all not working aliases.

instr-l.pdf last appeared in TL-2016.

IEEEtran_HOWTO.pdf and IEEEtran_bst_HOWTO.pdf last appeared in TL-2017. The bst file describes how to use the corresponding BibTeX style file.

--
Greetings

  Pete

Remember: use logout to logout.

**Comment by kberry** [comment]
1. AMS-LaTeX classes

texmf-dist/doc/latex/amscls-doc/Author_Handbook_Journals.pdf
(and other Author_Handbook_*.pdf)

amscls/README says:
    ---documentation files 2017/11/15
    instr-l.* removed; superseded by (class-specific) AMS Author Handbooks
    at https://www.ams.org/tex/author-handbook
    or http://mirror.ctan.org/tex-archive/info/amscls-doc.zip
And the package amscls-doc exists in TL.

    2. IEEEtran

I had to remove the *HOWTO.pdf files because sources are not
available. As I recall, the IEEEtran maintainer planned to make a new
release, but that has not happened. So no, there is no alternative.
I'd say just remove the special case, and whatever happens, happens.

Unless you want to start down the road of a new feature where texdoc can
resolve to a url and send it to a browser or whatever, instead of only
working with the local files.

    If the file `IEEEtran_HOWTO.pdf` doesn't exist in TeX Live, I think
    this file also needs to be updated.

I don't see why. The README.TEXLIVE files are created precisely when a
file that exists on CTAN is not installed in TL. Per the first line:
"The following files have been removed in the TeX Live installation ..."

Then it lists the files that have been so removed. --best, karl.
